controller/lib/core/sensors.cpp

```diff
@@ -107,3 +141,15 @@ float get_pressure_reading(AnalogPinId pinId) {
   // Sensitivity of all pressure sensors is 1 V/kPa; no division needed.
   return ((float)runningSum * ADC_LSB);
 }
+
+float convert_diff_pressure_to_volumetric_flow(
+    float diffPressureInKiloPascals) {
+  if (diffPressureInKiloPascals > 0.0f) {
+    return ROOT_OF_TWO_OVER_DENSITY_OF_AIR * venturiAreaProduct *
+           bernoulliAreaDivisor * sqrtf(diffPressureInKiloPascals * 1000.0f);
+  } else {
+    return -1.0f * ROOT_OF_TWO_OVER_DENSITY_OF_AIR * venturiAreaProduct *
+           bernoulliAreaDivisor *
+           sqrtf(-1.0f * diffPressureInKiloPascals * 1000.0f);
+  }
```

Comment:
Another advantage of getting rid of the variable initialization in sensors_init is that I think we can move a lot of the global constants computed therein down into this function.  Because they're constants, they'll only be computed once.  That avoids putting its logic into two places, which is nice...

Reply:
Yep!